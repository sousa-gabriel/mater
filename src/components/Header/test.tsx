import { render, screen } from '@testing-library/react'
import { Header } from '.'

describe('Header', () => {
  it('should render the heading', () => {
    // Renderiza o componente
    const { container } = render(<Header />)
    // Verifica se o heading est√° sendo renderizado
    expect(
      screen.getByRole('heading', { name: /header title/i })
    ).toBeInTheDocument()

    // ScreenShot
    expect(container.firstChild).toMatchSnapshot()
  })
})
