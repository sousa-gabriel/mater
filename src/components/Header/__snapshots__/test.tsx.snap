// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the heading 1`] = `
<header>
  <h1>
    Header title
  </h1>
</header>
`;
